### DISCUSSION TITLE
"Create and Add Another" error

### DISCUSSION THREAD

**Discussion by penrik** [netbox-community/netbox#12473] [Q&A] [answered]
After upgrading to 3.5.0 pressing "Create and Add Another" under  IP Addresses thows:

`<class 'AttributeError'>

'str' object has no attribute 'is_cloneable'

Python version: 3.9.2
NetBox version: 3.5.0`

Is it just me or is this a bug?

**Comment by candlerb** [comment, marked as answer]
Search "Create and Add Another" in Discussions, and you will find recent discussion #12414 which links to issue #12395 and will be resolved in v3.5.1
